taskfile.yaml

```
@@ -8,7 +8,11 @@ set: ["u", "pipefail"]
 shopt: ["globstar"]
 
 vars:
+  G_EXPORTS_DIR: "{{.ROOT_DIR}}/exports"
+  G_LINT_CONFIGS_DIR: "{{.G_EXPORTS_DIR}}/lint-configs"
   G_OUTPUT_DIR: "{{.ROOT_DIR}}/build"
+  G_YSTDLIB_PY_DIR: "{{.G_EXPORTS_DIR}}/ystdlib-py"
+  G_PYFIND_CMD: "uv run --project '{{.G_YSTDLIB_PY_DIR}}' pyfind"
```

```suggestion
  G_PYFIND_CMD: "uv run --project '{{.G_YSTDLIB_PY_DIR}}' pyfind"
  G_YSTDLIB_PY_DIR: "{{.G_EXPORTS_DIR}}/ystdlib-py"
```
Assuming we order these variables in alphabetical order.

---

Iirc this breaks since `G_PYFIND_CMD` relies on `G_YSTDLIB_PY_DIR`. I'll try spliting `G_PYFIND_CMD` into another grouping/category to make it nicer.